tsconfig.strict.json

```diff
@@ -67,8 +72,7 @@
     "lib/datasource/go/types.ts",
     "lib/datasource/helm/common.ts",
     "lib/logger/err-serializer.spec.ts",
-    "lib/manager/gradle/shallow/types.ts",
```

Comment:
why removed?

Reply:
It's `excluded` section, now the file is fixed